chapters/00.Introduction/00-GetGoingWithJulia.md

```diff
@@ -0,0 +1,153 @@
+---
+author: dimiboeckaerts
+---
+
+# Let's get going with Julia
+
+Let's get going with Julia! This summary-based guide into programming with Julia is meant for students and researchers with a general background in programming but no specific knowledge of Julia. The first two sections get you up and running in Julia, while the third and fourth section dive into some more (bio)engineering-oriented material. Finally, the last section provides additional resources to explore more from Julia. Happy learning!
+
+[TOC]
+
+
+## The basics
+
+What is Julia again?
+https://julialang.org/learning/getting-started/
+
+Why should you learn Julia?
+https://blog.goodaudience.com/10-reasons-why-you-should-learn-julia-d786ac29c6ca
+
+Let's download & install Julia. You can either download the language itself and engage with it through the command-line (or notebooks, see below):
```

Comment:
Don't know this. I think Visual Studio Code is also used a lot. Best mention this as well.